docs/how-to-guides/Deploy-FhirCOnverter-local.md

```diff
@@ -0,0 +1,48 @@
+# Fhir app container deployment for local changes
```

Comment:
@maryamasaaditeh The API for the convert isn't currently in OSS so at this time there isn't any way for a external user to create a custom image at this time.  Thank you for adding these instructions but can you please remove them from the OSS repo and add them to the internal Convert repo under the docs folder?  Thank you!

Reply:
done